crates/nostr/src/event/tag/kind.rs

```diff
@@ -125,6 +125,8 @@ pub enum TagKind<'a> {
     ///
     /// <https://github.com/nostr-protocol/nips/blob/master/C0.md>
     Runtime,
+    /// Server
+    Server,
```

Comment:
Miss the conversion from a string to TagKind in `impl<'a> From<&'a str> for TagKind<'a>`